srcpkgs/rundird/files/rundird/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/rundird
```

Comment:
I don't think we spell out the full path for most services, and it shouldn't be necessary. No need for `OPTS`?

Reply:
rundird doesn't have any options, I'll get rid of the unnecessary full path.